api/migrations/0001_initial.py

```diff
@@ -1,104 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I've seen it done both ways - constantly deleting and remaking everything inside migrations, and also trying to make sure that migrations are never deleted and always historically accurate. I think the latter is to keep production databases afloat though, so I don't think there's any harm in redoing migrations right now, so long as it's a complete redo!